lib/daru/index.rb

```diff
@@ -330,5 +341,11 @@ def values
     def inspect
       "Daru::MultiIndex:#{self.object_id} (levels: #{levels}\nlabels: #{labels})"
     end
+
+    def factor_out
+      # Remove levels not needed for display
+      useless = levels.index { |l| l.length > 1 }
+      drop_left_level useless
+    end
```

Comment:
Also `factor_out` is not a very verbose method name. Can you make it more specific? Also add documentation?